studio/components/interfaces/TableGridEditor/SidePanelEditor/RowEditor/RowEditor.tsx

```diff
@@ -111,9 +113,16 @@ const RowEditor: FC<Props> = ({
     }
   }
 
-  const onSaveChanges = (resolve: any) => {
+  function resolve() {
```

Comment:
remove this function - just updated line 144 to setLoading(false)

Reply:
You might had just broke something.. can't tell until i run it locally though 🤔